last_seen.py

```diff
@@ -1,5 +1,5 @@
 from brutal.core.plugin import BotPlugin, cmd, event
-from time import strftime, gmtime
+import datetime
```

Comment:
@Adman this is a clear nitpick but if you imported `datetime from datetime` you wouldn't have to use `datetime.datetime` everywhere (which doesn't make too much sense anyway).